building/build-debs/homeworld-admin-tools/src/iso.py

```diff
@@ -69,9 +69,12 @@ def gen_iso(iso_image, authorized_key, cdpack=None):
         preseeded = resource.get_resource("preseed.cfg.in")
         generated_password = util.pwgen(20)
         creation_time = datetime.datetime.now().isoformat()
+        git_hash = subprocess.check_output("git", "rev-parse", "HEAD")
```

Comment:
This is the wrong invocation of check_output. It takes a single list parameter, not a variable number of parameters.

Reply:
Also we’re not in the Git repository when this code runs.

Reply:
Ah, yes. See `version.py` for the correct source of the version information.